test/index.js

```diff
@@ -21,25 +21,23 @@ const testRunner = {
   addAfter(func) {
     this.after.push(func);
   },
-  run() {
-    Promise.all(testRunner.before.map(before => before()))
-      .then(() => {
-        this.tests.forEach(args => tape(args.name, args.test));
-        this.after.forEach(after => tape.onFinish(after));
+  async run() {
+    try {
+      await Promise.all(testRunner.before.map(before => before()));
+      this.tests.forEach(args => tape(args.name, args.test));
+      this.after.forEach(after => tape.onFinish(after));
+    } catch (err) {
       // eslint-disable-next-line no-console
-      }).catch(console.error);
+      console.error(err);
+    }
   }
 };
 
 Promise.all([
```

Comment:
Could be `const [jsonFixtures, ...] = await Pro mise.all()`

Reply:
It can't because it's at the top level of the file.

To do that we would need to wrap the whole thing in a function and then invoke it.

Reply:
Ah, yeah, didn't see that with github's hiding of the unedited part 👍